content/en/blog/2023/waypoint-proxy-made-simple/index.md

```diff
@@ -0,0 +1,221 @@
+
+{{< text bash >}}
+$ istioctl proxy-config clusters deploy/bookinfo-reviews-istio-waypoint
+SERVICE FQDN                         PORT SUBSET  DIRECTION   TYPE         DESTINATION RULE
+agent                                -    -       -           STATIC
+connect_originate                    -    -       -           ORIGINAL_DST
+encap                                -    -       -           STATIC
+kubernetes.default.svc.cluster.local 443  tcp     inbound-vip EDS
+main_internal                        -    -       -           STATIC
+prometheus_stats                     -    -       -           STATIC
+reviews.default.svc.cluster.local    9080 http    inbound-vip EDS
+reviews.default.svc.cluster.local    9080 http/v1 inbound-vip EDS
+reviews.default.svc.cluster.local    9080 http/v2 inbound-vip EDS
+reviews.default.svc.cluster.local    9080 http/v3 inbound-vip EDS
+sds-grpc                             -    -       -           STATIC
+xds-grpc                             -    -       -           STATIC
+zipkin                               -    -       -           STRICT_DNS
+{{< /text >}}
+
+Note the list for `outbound` clusters is empty, which you can confirm using `istioctl proxy-config c deploy/bookinfo-reviews-istio-waypoint --direction outbound`! What is nice is that you didn’t configure any `exportTo` on other bookinfo services (for example, `productpage` or `ratings` services). In other words, the `reviews` waypoint isn’t made aware of any unnecessary clusters, without any extra manual configuration from you.
+
+Display the list routes for `reviews` waypoint proxy:
+
+{{< text bash >}}
+$ istioctl proxy-config routes deploy/bookinfo-reviews-istio-waypoint
+NAME                                                    DOMAINS MATCH              VIRTUAL SERVICE
+encap                                                   *       /*
+inbound-vip|9080|http|reviews.default.svc.cluster.local *       /*                 reviews.default
+default
+{{< /text >}}
+
+If you recall, you didn’t configure any Sidecar resources or `exportTo` configuration on your Istio networking resources. You deployed the `bookinfo-productpage` route earlier to configure Istio ingress gateway to route to `productpage` but the `reviews` waypoint isn’t made aware of any irrelevant routes.
+
+Display the detailed information for the `inbound-vip|9080|http|reviews.default.svc.cluster.local` route, you’ll see the weighted based routing configuration to direct 90% traffic to `reviews` v1 and 10% traffic to `reviews` v2, along with some of Istio’s default retry and timeout configurations. This confirms the traffic and resiliency policies are shifted from source to destination oriented waypoint discussed earlier.
```

Comment:
```suggestion
Display the detailed information for the `inbound-vip|9080|http|reviews.default.svc.cluster.local` route, you’ll see the weighted based routing configuration to direct 90% traffic to `reviews` v1 and 10% traffic to `reviews` v2, along with some of Istio’s default retry and timeout configurations. This confirms the traffic and resiliency policies are shifted from the source to the destination oriented waypoint as discussed earlier.
```
maybe doesn't need the as

Reply:
Looks like i can't apply this...commit button is disabled.